content/posts/2025-07-17-whats-new-1-9/index.md

```diff
@@ -0,0 +1,228 @@
+```
+
+For more information, see [FLP filtering language](https://github.com/netobserv/flowlogs-pipeline/blob/main/docs/filtering.md).
+
+
+## UDN Mapping (GA)
+
+The eBPF feature **UDNMapping** reached General Availability (GA).  Network Observability added support for the ClusterUserDefinedNetwork object.  In 1.8, it only supported UserDefinedNetwork.  ClusterUserDefinedNetwork allows pods in different namespaces to communicate with each other using the cluster UDN.  To learn more about how to set up a UDN, see the article [User defined networks in Red Hat OpenShift Virtualization](https://www.redhat.com/en/blog/user-defined-networks-red-hat-openshift-virtualization).
+
+
+## Network Observability CLI enhancements
+
+The Network Observability CLI is a command line tool based on the Network Observability code base.  It is an `oc` plugin that captures, displays, and saves flows, metrics, and/or packet information.
+
+Installation is simple.  Download the [oc_netobserv file](https://mirror.openshift.com/pub/cgw/netobserv/latest/) and put it in a location that is accessible from the command line (e.g. */usr/local/bin* on Linux).  Make the file executable.  You don't even need to install Network Observability Operator, nor will it conflict with it if it's installed.
+
+You must be able to access your cluster from the command line using `oc` or `kubectl`.  It's best to widen your terminal to 100 characters or more for a better display.  To run the program, enter `oc netobserv` or just call the script directly, `oc-netobserv`, followed by various options.
+
+Decide whether you want to see flows, metrics, or capture packets.  If you want more information on this, enter one or more of these commands.
+
+```
+oc netobserv flows help
+oc netobserv metrics help
+oc netobserv packets help
+oc netobserv help  # for general help
+```
+
+With flows, it displays a text-based traffic flows table.  With metrics, you are given a link that creates a dynamic dashboard in the OpenShift web console.  And with packets, you also get flows, and it saves a pcapng file that can be loaded in with a tool such as Wireshark.  All of this data can be saved locally upon exit.
+
+Network Observability CLI deploys in its own namespace and is automatically removed once the CLI exits.  To manually exit, press ctrl-c.  It asks if you want to save the data in the directory **./output** and then exits.
+
+The rest of this section covers the new features in Network Observability CLI 1.9.  Here are the new options:
+
+| Option                    | Description                          |
+| ------------------------- | ------------------------------------ |
+| `--enable_ipsec`          | Enable eBPF IPsec tracking feature   |
+| `--enable_network_events` | Enable eBPF Network Events feature   |
+| `--enable_udn_mapping`    | Enable eBPF UDN Mapping feature      |
+| `--sampling`              | Set sampling interval, defaults to 1 |
```

Comment:
I see you specifically changed sampling ratio to interval, can you elaborate why?
Previously I wanted to change "rate" to ratio because IMO "rate" isn't very accurate (we tend to understand rate as "something per-second", right?)
But interval doesn't seem more accurate either?

Reply:
Awesome, I didn't know this meaning of "interval", I've been struggling to find an accurate term all this time 
So now we need to update the texts & doc everywhere 😄

Reply:
https://issues.redhat.com/browse/NETOBSERV-2355